open-ce/build_env.py

```diff
@@ -35,20 +35,12 @@
 import argparse
 import os
 import sys
-from itertools import product
-
-try:
-    import conda_build.metadata
-    import conda_build.api
-    from conda_build.config import get_or_merge_config
-except ImportError as error:
-    print("Cannot find `conda_build`, please see https://github.com/open-ce/open-ce#requirements"
-          " for a list of requirements.")
-    sys.exit(1)
 
 import build_feedstock
+from build_tree import BuildTree, DEFAULT_GIT_LOCATION
 import docker_build
 import utils
+from utils import OpenCEError
 
 DEFAULT_GIT_LOCATION = "https://github.com/open-ce"
```

Comment:
It looks like you're importing `DEFAULT_GIT_LOCATION`, but also define it here.

Reply:
Thanks! Should be fixed now.